src/net.cpp

```diff
@@ -514,6 +514,15 @@ void CNode::SetAddrLocal(const CService& addrLocalIn) {
     }
 }
 
+bool CNode::ConnectedViaTor() const
+{
+    if (m_conn_type == ConnectionType::INBOUND) {
```

Comment:
```suggestion
    if (IsInboundConn())
```

Reply:
(I'm testing building on this function ATM with the following in `CNodeStats`)
```cpp
stats.m_network = ConnectedViaTor() ? uint32_t(NET_ONION) : addr.GetNetClass();
```

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19998#issuecomment-701533052).